variables.tf

```
@@ -0,0 +1,51 @@
+variable "prefix" {
+  default = "tfvmex"
+}
+
+variable "resource_group_name" {
+  type    = string
+  default = "example-resources"
```

`resource_group_name` has a default of "example-resources". Because the task requires a manually created resource group, this default might be misleading. Either remove the default to force the caller to explicitly pass the correct resource group name, or add a `description` explaining that the caller must create a resource group with that name or override the variable at runtime.